### PR TITLE
Add methods to read prescriptions and download APKs

### PR DESCRIPTION
**Pull request by 0queue**
This code has been living in a fork for a while.  Adds `Sheets#fetchPrescription()` and `Sheets#fetchApks()`, for use by the frontend.

### CODE REVIEW

File: app/src/main/java/edu/umd/sheets436/MainActivity.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URL;
+import java.util.List;
```
Comment (MrPickles) [inline app/src/main/java/edu/umd/sheets436/MainActivity.java]: Unused import?

File: sheets436/build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ android {
         minSdkVersion 22
         targetSdkVersion 25
         versionCode 1
-        versionName "0.0.5"
+        versionName "0.0.6-prescription"
```
Comment (MrPickles) [inline sheets436/build.gradle]: (nit) Why the `prescription` part in the name?

**Review by MrPickles — APPROVED** [review]
Looks good to me. Just two nits which you can feel free to ignore.

File: sheets436/build.gradle (hunk 2 above)
Comment (MrPickles) [reply sheets436/build.gradle]: Actually, do you mind changing this to `0.0.7`? I forgot to update the version when I made the `0.0.6` release.

Comment (0queue) [reply sheets436/build.gradle]: Yes, I'll change it to 0.0.7.  I'll do that after the other PR lands

File: app/src/main/java/edu/umd/sheets436/MainActivity.java (hunk 1 above)
Comment (0queue) [reply app/src/main/java/edu/umd/sheets436/MainActivity.java]: Woops thanks

**Review by MrPickles — APPROVED** [review]
LGTM, merge whenever you'd like.

**Pull request merged**